### PR TITLE
Update lux and correct dependencies

### LINKED ISSUE #358: Update lux to v4.5.1
The version of `lux-design-system` needs to be updated to v4.5.1 to resolve the accessibility issue described in detail in https://github.com/pulibrary/lux/issues/424.

This can be done by updating the versions locally and pushing the changes to Git instead of directly making changes to `yarn.lock`.

### PR DESCRIPTION
**Pull request by ishasinha1**
Closes #358 

Added `webpack` to `package.json` because lux required it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
     "css-loader": "^5",
-    "lux-design-system": "^4",
+    "lux-design-system": "^4.5.1",
     "postcss-cssnext": "^3.1.1",
     "sass-loader": "^10",
     "vue": "^2.6.14",
     "vue-loader": "^15.9.8",
-    "vue-template-compiler": "^2.6.14"
+    "vue-template-compiler": "^2.6.14",
+    "webpack": "^5.0.0"
```
Comment (sandbergja) [inline package.json:11]: Does it work without adding webpack?  My understanding was that lux needs it internally, but sites that use lux shouldn't need it.

**Review by sandbergja — COMMENTED** [review]
Thanks, @ishasinha1 !  One question about webpack.

File: package.json (hunk 1 above)
Comment (ishasinha1) [reply package.json:11]: I think it does work. I was getting a warning, so I decided to make this change. I could remove this!

**Review by sandbergja — APPROVED** [review]
Thanks, @ishasinha1 ! 🟪

**Pull request merged**